test/unit/lua_msgpack.c

```diff
@@ -0,0 +1,117 @@
+#include <lualib.h>
+
+#include "core/assoc.h"
+#include "core/cord_buf.h"
+#include "core/fiber.h"
+#include "core/memory.h"
+
+#include "lua/msgpack.h"
+#include "lua/serializer.h"
+
+#include "mpstream/mpstream.h"
+
+#include "small/ibuf.h"
+
+#define UNIT_TAP_COMPATIBLE 1
+#include "unit.h"
+
+static void
+mpstream_error_mock(void *ctx)
+{
+	(void)ctx;
+}
+
+/**
+ * Checks that translation of first-level `MP_MAP` keys is done correctly.
+ */
+static void
+test_translation_in_encoding(lua_State *L)
+{
+	plan(3);
+	header();
+
+	struct mh_strnu32_t *translation = mh_strnu32_new();
+	const char alias[] = "x";
```

Comment:
Nit: This looks weird. Why not
```C
const char *alias = "x";
```
and use `strlen(alias)` instead of `lengthof(alias) - 1`?

Reply:
Discussed with @locker: came to an agreement this approach is weird and not worth it — fixed.